x-team/xp-ui
# Use AppLink name to accurate express the kind of component expected by the props

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3433

## Description

Instead using custom names, we decided to use `AppLink` for all similar places because that express that the component expected is the consumer application Link. That's a dependency from the consumer application and not available inside XP-UI, as it can vary for each case. For instance, on XP-Registration the Link is provided by Gatsby, while in XP it's provided by react-router.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3433-use-AppLink` | [XP Registration](https://github.com/x-team/xp-registration/pull/117)

## Steps to Test or Reproduce

This PR updates `JobCard` and `JobsPageBreadcrumbs` but it doesn't change any behaviour or styles, it's just updates the naming used for link props to keep consistency.

You can check the Storybook for no regressions in the link message (see `Actions` panel when click on JobCard title or Breadbrumb link):

- http://localhost:9001/?path=/story/ui-components-jobcard--basic-usage
- http://localhost:9001/?path=/story/ui-components-jobspagebreadcrumbs--basic-usage
